src/more-infos/more-info-climate.html

```diff
@@ -46,22 +47,51 @@
         width: 100%;
       }
 
+      paper-range-slider {
+        width: 198px;
```

Comment:
How did you determine these values ? The width of the dialog is based on the name